charts/brig/templates/tests/brig-integration.yaml

```diff
@@ -54,22 +54,29 @@ spec:
     # The other test, "user.auth.cookies.limit", is skipped as it is flaky.
     # This is tracked in https://github.com/zinfra/backend-issues/issues/1150.
     command: [ "brig-integration", "--pattern", "!/turn/ && !/user.auth.cookies.limit/" ]
+    securityContext:
```

Comment:
I think you don't have to care about any containers with names ending with `-integration`. These are only meant for integration tests.